### PR TITLE
Clean up, documentation, and updates to old code and `actions.json`

### PR DESCRIPTION
**Pull request by 00sapo**
For simplicity, I used `gpt-3.5-turbo` model, but davinci-code could be used in edit mode if prompt are changed as well.

I have also restyled the readme a little.

### CODE REVIEW

**Comment by 00sapo** [comment]
Note that, for me, `ChatGPTRunCustomCodeAction` is totally broken (it starts a popup asking for text, but it can't find any action)

**Comment by 00sapo** [comment]
Sorry, I didn't see your merge. I added some improvements to the readme and cleaned old code

File: README.md
Comment (jackMort) [inline README.md:1]: Great job!

File: lua/chatgpt.lua [hunk 1]
```diff
@@ -37,10 +38,8 @@ M.run_action = function(opts)
   module.run_action(opts)
 end
 
-M.run_custom_code_action = function(opts)
-  module.run_custom_code_action(opts)
-end
-
-M.complete_code = module.complete_code
```
Comment (jackMort) [inline lua/chatgpt.lua:44]: Please keep `complete_code `as it is. It's a different flow in the early stage, but I'm going to work on it more to make it work similar to Copilot.

File: lua/chatgpt/flows/actions/init.lua [hunk 2]
```diff
@@ -1,13 +1,13 @@
 local M = {}
 
-CUSTOM_CODE_ACTION = [[
```
Comment (jackMort) [inline lua/chatgpt/flows/actions/init.lua:3]: feel free to remove it, maybe it doesn't make sense as you said

File: lua/chatgpt/flows/code_completions/init.lua
Comment (jackMort) [inline lua/chatgpt/flows/code_completions/init.lua:1]: please revert this change as I mentioned before

File: lua/chatgpt/module.lua [hunk 3]
```diff
@@ -15,7 +15,6 @@ local Sessions = require("chatgpt.flows.chat.sessions")
 local Session = require("chatgpt.flows.chat.session")
 local Actions = require("chatgpt.flows.actions")
 local Tokens = require("chatgpt.flows.chat.tokens")
-local CodeCompletions = require("chatgpt.flows.code_completions")
```
Comment (jackMort) [inline lua/chatgpt/module.lua:18]: same here.

File: lua/chatgpt/module.lua [hunk 4]
```diff
@@ -307,7 +306,6 @@ end
 
 M.edit_with_instructions = Edits.edit_with_instructions
 M.run_action = Actions.run_action
-M.run_custom_code_action = Actions.run_custom_code_action
-M.complete_code = CodeCompletions.complete
```
Comment (jackMort) [inline lua/chatgpt/module.lua:311]: same here.

File: plugin/chatgpt.lua [hunk 5]
```diff
@@ -29,13 +29,9 @@ end, {
...
 })
 
-vim.api.nvim_create_user_command("ChatGPTRunCustomCodeAction", function(opts)
-  require("chatgpt").run_custom_code_action(opts)
-end, {
-  nargs = "*",
-  range = true,
-})
-
-vim.api.nvim_create_user_command("ChatGPTCompleteCode", function(opts)
```
Comment (jackMort) [inline plugin/chatgpt.lua:39]: same here.

**Review by jackMort — CHANGES_REQUESTED** [review]
In overall it looks great, thanks for handling that! please just revert your changes related to code_completion I'm going to improve that to work similar to copilot

File: README.md
Comment (00sapo) [reply README.md:1]: Thanks, I applied the commit you asked for

**Review by jackMort — APPROVED** [review]

**Comment by jackMort** [comment]
Great, thanks! Please just fix the style errors to pass checks and we are good to go
